README.md

```diff
@@ -95,8 +95,15 @@ orchestrator.registerScenario('proper zome call', async (s, t) => {
 
   // and install a single happ
   const carol_blog_happ = await carol.installHapp([dnaBlog])
-  // or a happ with a previously generated key
-  const carol_test_happ_with_bobs_test_key = await carol.installHapp([dnaTest], bob_blog_happ.agent)
+
+  // or a happ with a previously generated key and previously registered dna by hash
+  // which you can get from the cells as returned in the InstalledHapp from previous install
```

Comment:
I find this sentence hard to parse.
```
  // or a happ with a previously generated key and previously registered dna by hash
  // which you can get from the cells as returned in the InstalledHapp from previous install
```
Maybe I can rewrite it once I understand how the feature works...